addons/advanced_throwing/XEH_postInit.sqf

```diff
@@ -10,6 +10,9 @@ if (!hasInterface) exitWith {};
 // Temporary Wind Info indication
 GVAR(tempWindInfo) = false;
 
+//Enable/Disable grenade preview when preparing
+GVAR(showThrowable) = true;
```

Comment:
```suggestion
GVAR(hiddenThrowables) = [];
```

Reply:
Maybe only set it here if it's nil?
A mod might add to the list earlier

Reply:
should just move define to preInit